upgrade/helpers/rhevm.py

```diff
@@ -275,18 +271,18 @@ def validate_and_create_product_templates():
             execute(check_ntpd, host=cap_host)
             execute(katello_restart, host=cap_host)
             try:
-                thread.start_new_thread(create_rhevm_template,
-                                        (sat_instance,
-                                         cluster,
-                                         new_sat_template,
-                                         storage
-                                         ))
-                thread.start_new_thread(create_rhevm_template,
-                                        (cap_instance,
-                                         cluster,
-                                         new_cap_template,
-                                         storage
-                                         ))
+                execute(create_rhevm_template,
```

Comment:
Execute needs host parameter to pass. I see that is missing. Otherwise you can only call ```create_rhevm_template```' function without execute method.